ledger/sandbox/src/main/resources/db/migration/recompute-sha256sums.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euxo pipefail
+
+cd "$(dirname "${0}")"
+find . -type f -name "*.sql" | xargs sha256sum | awk '{print "echo " $1 " > " $2}' | sed 's/\.sql/\.sha256/' | sh
```

Comment:
You could avoid generating the script by doing something like:

```sh
for file in **/*.sql; do
  shasum -a 256 "$file" | awk '{ print $1 }' > "$(basename "$file" .sql).sha256"
done
```

This is basically what's in _ledger/ledger-on-sql/hash-migrations.sh_. Your call though.

Reply:
Mh, I'd open a ticket to extract this and put it in a common place, makes sense?

Reply:
Sounds good to me.